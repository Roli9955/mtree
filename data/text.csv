id;val
0;gxkiactblx
1;nqdpdplhiq
2;ppbaidqlgg
3;bjvsqddxgh
4;fmdxauhrjk
5;zpqvshezrk
6;hthgoasrjh
7;szcdhgmihn
8;pkzpnulwpr
9;steuvmdnsz
10;hhebbjfvgm
11;xfwrndggsn
12;qldsualydl
13;ppinfgtxzt
14;pifjkzwgkh
15;oyfipkopnu
16;kpbrfbmpap
17;kuikigkvkx
18;hlqfnkngur
19;zyixjnaygl
20;puzongegvg
21;ovfkgubcdg
22;hnteambcyw
23;ccmfuqowio
24;zsihjxxypf
25;suxfqhzcjt
26;uvtefbcngn
27;riptvfaekq
28;dpnsbiwlkl
29;hkdmgoueca
30;bmhogpepvw
31;nrkbkfqzef
32;qwuznqljyk
33;btbyuvpnly
34;jzgmljrixo
35;jtidflyame
36;zvcpbqqymf
37;phladajdbo
38;jcqwndejcs
39;vfcicbajao
40;cbvrkhgqoq
41;dpvzxuhnwt
42;qbvlmozsan
43;kljequatli
44;otyplcblqv
45;cgyenwmsmj
46;ecdvvekeob
47;xzucjqxjno
48;zwrqrgvavu
49;wvtpmodwyu
50;xrclelqzya
51;dsfgejnkrv
52;dpiphdmvur
53;zeegzcbgwd
54;btgcwqyrhj
55;sldbjeoaqq
56;zjophxeowd
57;ekiupqazfd
58;wmcutusniu
59;zulhycujsm
60;ezwwrbmokh
61;gvnlwsmwvx
62;mxyeepmthj
63;frehbmojhx
64;zhuagsfowx
65;lugwvgewme
66;hsgxrkdkel
67;vpvyivotix
68;dwvkmzemyu
69;cxuppplvds
70;budujmngcr
71;yhgtpdnvta
72;nbipgzyhne
73;sygcaaixid
74;thoprmvwje
75;anzsmikjfy
76;xvcwrjpkqm
77;kahnmsruir
78;utugpfvudy
79;efzpsrjgwc
80;applsxsdzg
81;rsnmxrzhey
82;bpswfsosdy
83;nwqsvldkpc
84;gnasdxzwpw
85;mpwdaurxzh
86;gpmwxgxqgg
87;tifukkfdrl
88;gsporkvfog
89;ozfgnhsfmh
90;rkrcpyabli
91;qeehaluemt
92;jntljhmhmk
93;eadyppgudl
94;pjgapssifr
95;bsvzrattxl
96;ujdacephwm
97;nodccuxqht
98;tloezorvin
99;tiwgoicjin
100;pqetnmttbm
101;iycvabrpnp
102;mymjkaqyzw
103;iwidwjbggd
104;iumjeuigix
105;jzuhdsapkw
106;iwgcswgiqm
107;woltzyitic
108;sjaydknbyy
109;rrlvjfzjyg
110;ujrymkbsrs
111;izavkwgcxq
112;liukixdymg
113;nkncrihpmz
114;qreajxbdsj
115;ogclkgowoo
116;mudhmulere
117;ruazmlutcr
118;szaxfvaivn
119;efbdxezgfq
120;biwubgleyj
121;pccwaxqccq
122;otmxtswvms
123;ohmhjsqkjo
124;cvhxlxokdj
125;qojirxirgs
126;uyqtoxpoxv
127;ojlfcbirlq
128;zmyzsiqzwz
129;ojwszkuwwx
130;kzldnrxpez
131;lrwdvwzfgv
132;wtqlckvvuv
133;wvxcqnkgkd
134;prsnyshdmj
135;dfgrwyjrbf
136;hrwgjymydg
137;ansexcqcoy
138;vwkucefijf
139;ziznjgyyix
140;dqbkotuivz
141;emnzwrxaiv
142;vwhgajpdug
143;yfxobnkibs
144;fqnziajfwa
145;wjobigavyq
146;stqaehkowm
147;myhwzyuoac
148;xacljjkkch
149;qcqgdsaokd
150;qcwibdxaku
151;dufupbhxdu
152;pjfvbygxux
153;bubyerqxyr
154;ilzdniofvt
155;kerfbylrxl
156;npksxbuklb
157;xmsnrjgsuf
158;ifdzgpdwjg
159;arrxahrroi
160;buhgvughky
161;yypjdmtksx
162;zpefzybiur
163;vgogrughdz
164;unzrbflsfp
165;nerbytouck
166;abyduwtmrw
167;ypbvtyrnup
168;zhfuovpwja
169;axrtavnkid
170;bsxwzwwvoh
171;ieqvqzplan
172;kkcvuwwewn
173;nzjghbbxox
174;jazqxssbtw
175;tfpgbxsmpm
176;pdlyqledpy
177;zgphamxkuu
178;nbwezfcwyl
179;czqtmzhvak
180;bzcydmvvzb
181;yxgkijkjmx
182;fejjrlrugx
183;ujudmjbbet
184;yfxgnkhwyh
185;wjiflncuxv
186;fefnlwnbrc
187;tsvrukbcmf
188;llxreqmmmw
189;vesqzxrwng
190;xldzkqltyv
191;nzptqjfpiq
192;wyfgkdlrrz
193;ysiozmymio
194;sphinncybd
195;almhyhxzpp
196;pjzmhqruqq
197;gpsfsshwzx
198;agsambuqyu
199;woxycgdvsc
200;ekvhjvjgtb
201;wgkouhhasl
202;hhpiafkdwl
203;yjyrwajlrk
204;fezicisdek
205;jszeghqmep
206;wgkedhldrm
207;npoyussder
208;wahxifpdld
209;mjgrdsojmk
210;pygfedpxpx
211;zfldliswfr
212;uburxdtvya
213;qswkxjwvtf
214;wpgsnffvpg
215;dvcauxvmom
216;kxvqengtav
217;hzvkegpnwo
218;wqoustllpg
219;ubydueatbx
220;sckazcddkp
221;bkiodxjqnf
222;htgrobqfqk
223;hrfybcxdqs
224;uubluynjiy
225;gshqrnpoqe
226;ebfhqczovj
227;ayxgsiexqr
228;dhlldfgjcz
229;pdpzmwxfeg
230;hhedtohmzx
231;jkxilixfoi
232;lrrpiigkmc
233;lkpimytjiu
234;vncoazrhri
235;zzovmmuzdu
236;qftrmxfgdu
237;pjkepewncc
238;qnxeravmcq
239;pbapjvjqgu
240;nslcxvuejt
241;vrpilwzbmn
242;tfrrrzuldy
243;thovugbusc
244;mvzilpxzss
245;dycqevedcd
246;erohkvpqlp
247;cmkwunidcf
248;jgjrximtni
249;gjpnwlkpzx
250;wnqhipdbqn
251;oilwjalwjk
252;zfhwhbdybt
253;lvwefilsiq
254;xjhducxlow
255;reuqfizxbm
256;pungoqbqht
257;zuwixbeicc
258;iryrfegbcy
259;noilhrbgad
260;dxjkukezfs
261;gsezedtuhr
262;emzvazvgdv
263;lyflehtlpp
264;lzurlnwwxj
265;ewuwamszyo
266;lfnseteunh
267;reatbeaedx
268;xfumasuebx
269;koipmfkomd
270;hdljodgfbh
271;asziqlykqw
272;wjkhxdcxvy
273;rbvlcqdczy
274;gmzzwdwlgn
275;wmmuwbyjdi
276;vodjlzeosm
277;vlcrmzxcyr
278;hvwmkyukmw
279;dnwrcmkoqu
280;izfpqdfray
281;lnnofekkcd
282;dnhhczshnm
283;yljobffzwv
284;taggunuoxl
285;fsjbwcwbll
286;ypuirbeqfs
287;uvhetkxaud
288;exhlyssfbf
289;tpnezkxpym
290;ardlpeenhu
291;ibteuhquoh
292;xdwpkbbpcf
293;bkdxjmmwjg
294;nuhqxjrstt
295;yljncrnaun
296;cbhbiwrozx
297;ousekssisp
298;kglppvuwvu
299;vczttiufgg
300;reunagycwi
301;wsugwhkryg
302;dtilggknbx
303;imfdjjfcdt
304;xretmxdcwt
305;azaqxpifig
306;cqevxytjuf
307;tyuivrhcum
308;tqtqzuunmc
309;gcqyfndudh
310;cbqeqzlbme
311;zdnddqcref
312;gettgyjaht
313;nsdajkahrf
314;ssapuscvdl
315;eqhsmaaxne
316;osezgtzywp
317;gljsejedpy
318;xsddnaylcy
319;egkerfkxvk
320;jfsyiajflv
321;xlxzvnfxep
322;tyefmqinuh
323;mupgdresvn
324;kiarqlbwjj
325;ekilbnaakp
326;tmgfhpigkd
327;ejidtqhhwi
328;ziuzidziwu
329;zbgpeeclqm
330;kxvrcqrmhe
331;hbbzqdssha
332;ditcocvtan
333;ukmzwqqkhj
334;oesvazbsix
335;bxghlaqgxk
336;fngxjlpuan
337;qcdpkgcnpa
338;yallskpmxc
339;wwamklqebi
340;rfljsypjtf
341;ogjetpzeth
342;lmhbdfzvsl
343;vujdfiabtu
344;fiqfuzvdqg
345;gkeusvvwqx
346;lerascwszb
347;bvqovmqypq
348;ezfxkfxvnc
349;zhpwbwjmzv
350;ncoqvufpch
351;hnoxnybpqc
352;kiagtytkxa
353;sqqlpzjwod
354;kqxidcmghw
355;onkdsullkj
356;xwgjluippu
357;talhipjtuk
358;ezgaxdthmf
359;xrsyolptmb
360;jpbvmnqgcd
361;tthssrrfrl
362;ysxhfiucic
363;slgimixjkz
364;svgirmzvto
365;zodzmktkwp
366;kfjtzfbdvv
367;zelejqpxry
368;gcklorntoy
369;txhwabwowo
370;ehveevrwbq
371;vnhlyxsaxu
372;qmtbyplmgg
373;ypvambomys
374;nwgtybfaub
375;rxwcartfsz
376;fbgxwizkbk
377;oeupulloyl
378;hvqcsuboik
379;jxmauyeuoi
380;hclwbpnwep
381;heomztgxir
382;bcvgvauuoa
383;tnffpgrdqn
384;vrsufjdpkw
385;tcxxfijauy
386;mhczwiukij
387;fvpoeizyor
388;omdemefffu
389;jybmvfrdgp
390;imodxpaqib
391;knqkiwsizp
392;mrijjwztpt
393;zsasslqbml
394;dcwtvigtqt
395;hvfjzufztn
396;vutegvitdn
397;whxhuyxwwb
398;qosoqsotlo
399;zeqftqamnm
400;wodewmlbcg
401;poxmexgtqp
402;lqcxxhsdxl
403;xvkdjkgggw
404;geudsgysfp
405;ggkbadzshh
406;qnbxfhjekk
407;fzmbnjhdft
408;gfyoiwpyjs
409;eymcvezzzs
410;ntjdstkjqm
411;qnqdutaxab
412;sruxrwofur
413;vieyzkiorv
414;bsrvqlxnvm
415;vimmvdbsvb
416;ktfvbeccvm
417;dpryjloury
418;mjyoijmbfc
419;rwjbzgzxhm
420;imxscmjjhg
421;vbvvzmifco
422;ycjjvbhsxe
423;gxgjkkrdtk
424;rqcwfezjhx
425;zpkblqmtat
426;sogmsefeii
427;wvlnhlxeuo
428;ussqlkajwd
429;twzynbgynq
430;xjbjmlulzt
431;uumltdeikz
432;cadqlhcbqp
433;folugzltyd
434;mwbnkhfrdn
435;ejxhehkqra
436;rqhapdnjli
437;yayhtomoft
438;ogsbrhsico
439;akwqzvkabj
440;zummswmmqf
441;hwlvcgvwki
442;byfaihwues
443;yefmkwaqul
444;qrazxrfgte
445;dpotvfhjus
446;riseifymgl
447;ouyxqhplxi
448;tangjvemoe
449;eiofmgsxnc
450;lzqszodefh
451;gqnjcozioy
452;gfrvlilgkt
453;sdhonwfsnj
454;ioypghaltc
455;ogopobxpus
456;mdmbwugibm
457;pdivjpcilx
458;nvduxwrdte
459;rxfmiohlui
460;crmpfhezis
461;tkfrwomkwz
462;pndrieaydf
463;txhrthshkg
464;swljyyczzz
465;nqyloiuxga
466;tdvheerxnb
467;jngpljjpvw
468;whsgwgkfqw
469;vgrlnikhyf
470;fykmaqqymd
471;lpowflxunm
472;uwihvaebjw
473;xdtgbevnqj
474;opxscnwfqg
475;zpcgcxdaap
476;jresilbtmc
477;ipiubqqudl
478;yjnrddnbrx
479;fglbarqyej
480;hljjpvyrem
481;dhuprwtcmd
482;bhprbaucwf
483;qxxxcmvypa
484;hefrjcrens
485;gwfyeynjgf
486;sddshcrtrc
487;ehoungkymk
488;iofsmtoshn
489;ydfkpywasd
490;ldhnvaqyrf
491;ssqvqpakkl
492;uunsronszk
493;byqneekvhm
494;icumroxqgt
495;jidsmdmhup
496;upcyxpqbtg
497;qlvkhhenuw
498;byoulnxuvu
499;bjoywkngev
500;vgyvrywpzc
501;mjkjoftdbt
502;csgpetfhah
503;icntebphnt
504;nswftqbyoc
505;kjiazpupse
506;lwpbconbda
507;scbouhxdja
508;zxihxrrntm
509;hldzhrwmjn
510;gxyisgzecx
511;gadyctjfzv
512;svlibckiyc
513;drfixtfszy
514;gtmcwdpmkn
515;tiubqzkvst
516;kxkmjbbeyq
517;qjraskaglo
518;oktqnowkpz
519;aswctlxocs
520;spzeqguexi
521;lmpfrzqykw
522;uhbvktsnky
523;abfyjtkujr
524;vrbsspxymv
525;glfohccowt
526;vjymqozgmn
527;wlaqoypbxp
528;votbbotshi
529;iqlmwwzvxp
530;iussukoqok
531;wvrrhtftwu
532;dyfywcecgc
533;tniexvvbfb
534;zggwwzwomv
535;zpwysoaeec
536;gtvkviucqv
537;qsixesicam
538;fdemiulamr
539;nomtnvzobe
540;rmslscklkl
541;zafcyjagyd
542;gnenyznvff
543;hkbimlduib
544;ioxqxxzbzk
545;sppiyalhik
546;deiiwykclf
547;axtgtceupx
548;xpiykvautr
549;fmseruqddd
550;txvdhqkara
551;kawrzgilhy
552;vvgmiqzoou
553;gqvxjoanzt
554;giywhodxvp
555;ycfotmlrhh
556;sttdqjhdtw
557;rtgbuekqqg
558;pwmoplvtst
559;rpkaglrbev
560;ajigfviqqg
561;czftrfhszf
562;askbeaxlgr
563;gyxhvgejvf
564;crzbkthsdg
565;pdkebkahrl
566;koctshsliv
567;uhywcgcdlx
568;xaghcknbea
569;qswtnmnhvc
570;npjhjdeifq
571;sgiggxwbsh
572;tnlsiiswja
573;nsxvlurngz
574;cjofjdazko
575;oxenqtvles
576;dplqgehoik
577;kssyodeseg
578;ynpemhguur
579;myyjxoxmjn
580;uzwztvbhkk
581;yvqvdosqbs
582;uygnqufehw
583;eazzzlvrsi
584;evjljycxlb
585;ncbyejxwxn
586;heucjxwyvc
587;soofijjdsg
588;pcxonbtbzn
589;afkhcsmxrz
590;bttmrlvhzy
591;pncanzozat
592;zllxivaxhn
593;lezehhhjio
594;cwdurxapoc
595;nayluonisr
596;vjvikzgdao
597;ndmbnjsppd
598;lgkxveghtu
599;nmsqbpviwo
600;lqxrtuiycq
601;zdknbfrpyl
602;vppqckrlfm
603;bkipyuppfk
604;axfjtymkrc
605;zpubsqrvup
606;vjljcpirde
607;szkgwrxxbk
608;sfcfcpaswl
609;nsieeopnju
610;lrrdvtfacz
611;tccyzargrr
612;lxpsxafwdx
613;gddihxaxdt
614;hczdccajoz
615;ziprurqjvl
616;wrkhhbixqi
617;nkiiuqjpug
618;ousjznsfke
619;cnjhetksfa
620;xhlhyyaeml
621;miuxjiavuw
622;cupuqnbgtw
623;bankmltnky
624;siuktnstda
625;ygxbazcblz
626;rakbdgwqnu
627;bdmakjvtxz
628;qyiygejuta
629;mzusibycmf
630;pfrfzjegpv
631;gwluauvqke
632;amszebfmyz
633;atzdgrtzyc
634;xumczetlbx
635;rgqrguoemb
636;tqboipxbki
637;klpxsbuhrr
638;ciffhkmqek
639;pajcowangs
640;argpirmssg
641;uzyoruoqye
642;sidxeapisx
643;bxbkgzgaay
644;cpuzerilza
645;gerjymfhny
646;xgvfjvqbgb
647;rhizqpweqs
648;yvavwsukdn
649;spysumbijk
650;orxlsvjtjq
651;hmasvxxrss
652;eirpvqyolm
653;duflwkuxxy
654;ttsgnjlehe
655;fyjmrkwxuo
656;tbgziomrdg
657;wjfknrhrkk
658;qdllirfzes
659;xgmglrugyz
660;lffvbytuyy
661;vwksbmfavm
662;whpzfwsasu
663;xtiepfnspl
664;kbrhyptypa
665;hrwmwcapkp
666;ulkdogpcpc
667;lpaidmrwxu
668;eduhqwkzsz
669;csnqazxczu
670;xgnrvnsieh
671;sdzxmonlkn
672;fwjpmdntzf
673;aimzezmyqp
674;wpaubbtiyc
675;vzmyceczaf
676;zzdhmsximm
677;jymzbwkryn
678;kyyxcerska
679;cfeirbrdej
680;nxezwielwt
681;ntjepyfnnm
682;vahdkxkuyc
683;evibkvtore
684;qiayfkcdxm
685;aagkkhoedr
686;ejxgbyadzu
687;acnclumrje
688;msiuwopgmb
689;pmumietuyx
690;yciwgwbule
691;wucsjvkvow
692;bzqzscctvg
693;kcejgjtoxu
694;otkilsegir
695;eqskgdbhqy
696;jtphjjfuks
697;aflbsixsfw
698;owhlbedkxw
699;mphswoadyu
700;gnpjugcahu
701;msxurjjejx
702;bicuimadnb
703;jjyzfgyndi
704;rhjyeirqvz
705;utzqibtgbi
706;bcyzlepctm
707;xcatumjxam
708;cdfjsalyof
709;rdusycfjqj
710;qbtleewemu
711;skafqvkpoo
712;rilvhsyxms
713;oziokabpie
714;nscvhkfsga
715;gonwnvdfmt
716;rbvopkghfo
717;wdrjokjgop
718;fsczjsevay
719;xrpmfahrsp
720;wgazujwyaw
721;mwahrtenip
722;cfumlchpkt
723;ctsbsmbghv
724;nbateyhidb
725;mvkczgjgsr
726;smrgfucgzd
727;iwzgmxhmdd
728;wykkpbsmia
729;etulpqxgkv
730;tqjcwlquob
731;hppkiiucpy
732;kdsdxggdgd
733;nagkgaurjl
734;ajfjxuvvqv
735;cypthxgxdv
736;sjshhmxrkw
737;axvflsghdi
738;amatjslfld
739;ftbhimxxyb
740;wombsfqnqr
741;ojreoibrfd
742;gproyuumip
743;iaihdzwrpo
744;qkdaccwoqm
745;lweatrqvmn
746;orcljiaecd
747;podurzgtlr
748;umprqlhxoh
749;ujfhtiybbs
750;jagglrocqn
751;urxrhpvbmf
752;vwdltdnekq
753;qtusnmklob
754;pktzpdthfp
755;swosdzwjto
756;kljjnnnpig
757;uznteqavmb
758;xhktyunmna
759;vczccpxqux
760;pqksvhjpcw
761;tiaywyhqvr
762;rotrtkkhji
763;zrmxmnxeap
764;goepeerelu
765;ticatsfhbr
766;akbxpqehgn
767;ihqiybzbhy
768;hinqozzkpl
769;mtefqfzrfp
770;uidyqloiaz
771;xsmckhsomp
772;loeeyuboqi
773;awnbbsmiva
774;ieefynwxlr
775;ilffxehvip
776;eqmcnefvhm
777;yemiwdlgpz
778;kwubksjqdj
779;hysazhsotq
780;cmimmrvaqh
781;ekrqhrawmq
782;qakwejzzuu
783;xcqrettvsc
784;xkjtpnssyk
785;owktzjplaq
786;fqzfkgozta
787;wghuetpxru
788;cglkxdlznz
789;wiwnwdfjzf
790;kelcbdfego
791;qruwiuwedx
792;xwxzsclnap
793;gyctlncavf
794;slhonijnhn
795;zgmcarlnsj
796;sabvkcvymv
797;uhhudaedhu
798;pxnsokteqo
799;pgijelhone
800;vyetjxqaat
801;dblblegsdy
802;ybpvfqsifn
803;cuwdcqqqvl
804;kwvkknvvpo
805;todyamwkex
806;qbfsevhbxz
807;crhkwwjflp
808;bgeimxgfwi
809;ucgdnfbgcj
810;plbpnpjccu
811;mibfrilwcx
812;xmdpxjojfx
813;omzzwcrjcx
814;egmvjgffxz
815;zvbddltvmy
816;bqrvhkwnax
817;srtjvuvdjc
818;byhoayuqbi
819;jyehesvnfl
820;efmksvdlym
821;solyymtxjv
822;gkevxgxduf
823;mwgdlzbxgx
824;ixeetqvzpi
825;pxmwnlrwaa
826;imnvwabxwb
827;qfekhxdhoy
828;nnhvhqltoq
829;rlkmlrsbbc
830;qxriawdaen
831;sriajdrmbj
832;gxtjtvdwuf
833;ibulusltch
834;zjjnxkhwpc
835;yvadrhrctx
836;gkfecnbwqp
837;bnubgfmnyo
838;uotlecugxq
839;zpipqoevai
840;bedixkmdiz
841;qhphkqkabf
842;hjiahtirop
843;xlhjfufech
844;oauzvbfjcl
845;lskbsajggo
846;detavptrcf
847;zpvliwcdhi
848;dvjqbgnauf
849;erdfrtmvxu
850;ckweevrsir
851;srglbzncae
852;ximxhljprs
853;pwglbojztj
854;hkxkcylvym
855;icrfnndbfo
856;gbltzycdbk
857;bdkzqbovwv
858;ivpypkbpcm
859;hohpzsebnt
860;qjzxwtbiuq
861;sepppctizj
862;dzafzgwraq
863;blrvmpznea
864;bbdkxkeynq
865;pixuowjsbt
866;stdkqpbsbe
867;ysbewnahyx
868;tmgtarqqey
869;yadgemtkrq
870;tlqvcguxak
871;pfsobnqxqp
872;drvrzabllk
873;nueyypcvjn
874;ibrlulzfva
875;goxtfzbgcg
876;kjjtxmmpxe
877;biedfxzzcp
878;kkiqohvgfy
879;isuvtjjjxd
880;ktjfknlhii
881;dysywvwqqk
882;zlnwynoizp
883;otwhumztuo
884;facjxtqmpd
885;edpoglhzyc
886;qpmilhjuex
887;lhigcnywpi
888;toghhzlcto
889;wfhlrrheuj
890;bedhdtfpty
891;dtiqbrvirp
892;invdqxqhfl
893;ghwfxakyka
894;kolchetuic
895;kepwmhxqli
896;mywjuqqvcy
897;kydpgknoxm
898;ozyrlsjenq
899;fffysmuyfg
900;kkqhzqgkvi
901;nkinpzhrcp
902;foaxmknuky
903;gwgzeedlpx
904;vuojlrbqwa
905;iusmnpaiaq
906;sxrmnljalf
907;qbnbhajqzk
908;kknhquoysw
909;vkdzzqosmw
910;tmyixhfszw
911;jsiojkhfrb
912;nwpqwaxcmo
913;uiurfokdho
914;rhfdnwrhfw
915;nrcmpgiroi
916;ycssgkplfj
917;nakxsinnha
918;kldnpgzzas
919;evjqxdcwol
920;hqlmzzcfgz
921;ggmkvaoowj
922;btmkaypjrw
923;memvigwdkd
924;zfrsmveuxl
925;xkpewaxluw
926;zvuocvrpyu
927;awtittlmuu
928;qzsqbtfuiy
929;bkucueemmc
930;iupbbkcibs
931;otqtcrqabd
932;hoxtlkyjfl
933;mrfdgsaunj
934;dbkmbjajnb
935;imvsfhtbfi
936;kiudfuizau
937;wbmjmmokel
938;bndhiphphk
939;hlzkqtezej
940;pexeigbyfe
941;tpditnroxq
942;mjsmbvflxp
943;xyyknnlbwi
944;mlsufsnbmz
945;dswsdcoxwn
946;lypamtfxhf
947;vmxryifady
948;kvhglholta
949;pnaczyswhf
950;zvejpsxsih
951;xvjjtpyhjl
952;huxrikaiql
953;lovthyagum
954;lgyxlicewf
955;zdceujwouk
956;ndckhrfptu
957;xxypwxhxix
958;qdqpbqgvja
959;zdincgsril
960;dojfslbmis
961;znbcbvdoxl
962;wyfegtrtve
963;rmokxhojkc
964;pgkxjevodm
965;lvrigfefdo
966;ftxxmrmaoh
967;zjqypdsays
968;tmsqpvoabs
969;fezfhlbqmm
970;soruzlvaih
971;gatnujzyzd
972;apnyivsbbw
973;atssmsnvml
974;ymypulddja
975;hktqlunwwu
976;elbhdsodcw
977;sknjhmnptd
978;kdcsukcbuv
979;mnczpajduo
980;qoogpchmhd
981;gguuwpwhgi
982;wboxecydyf
983;nqueohzbok
984;nctzebjbyh
985;yefvjabkkm
986;eibpmdoukx
987;jesskwjnua
988;avbcqkqmdk
989;cmuukvvjxf
990;fiqcmwavdz
991;vjcbehknun
992;qrzplwlvgn
993;gnizwnfwbu
994;yrvxuwmzbh
995;woxnkisjzb
996;morekedkzd
997;nakbzsmtny
998;rezmuzrjgv
999;dnkxdyenkz
