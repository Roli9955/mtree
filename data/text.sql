/* contrib/mtree_gist/data/text.sql */

SET client_min_messages TO WARNING;

DROP TABLE IF EXISTS TEXT_TEST CASCADE;
CREATE TABLE TEXT_TEST (
  id  INTEGER,
  val mtree_text
);
COPY TEXT_TEST(id, val) FROM '/home/postgres/test_files/text.csv' DELIMITER ';' CSV HEADER;

DROP INDEX IF EXISTS TEXT_TEST_IDX CASCADE;
CREATE INDEX TEXT_TEST_IDX ON TEXT_TEST USING GiST (val mtree_text_opclass);

SET enable_seqscan TO OFF;

SELECT COUNT(*) FROM TEXT_TEST;

SELECT * FROM TEXT_TEST;

SELECT id, val, (val <-> '0') AS dst FROM TEXT_TEST ORDER BY (val <-> '0');

SELECT * FROM TEXT_TEST WHERE val #<# '0';
