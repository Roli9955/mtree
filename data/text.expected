SET
DROP TABLE
CREATE TABLE
COPY 3
DROP INDEX
CREATE INDEX
SET
 id |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               val                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
----+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  0 | 0
  1 | a
  2 | zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
(3 rows)

 id |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               val                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | dst  
----+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------
  0 | 0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |    0
  1 | a                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |    1
  2 | zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz | 1024
(3 rows)

 id | val 
----+-----
  0 | 0
(1 row)

